templates/csv.yaml

```diff
@@ -398,7 +398,12 @@ spec:
                   # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
                   image: registry.ci.openshift.org/openshift/openshift-serverless-nightly:knative-openshift-ingress
                   imagePullPolicy: Always
+                  ports:
+                    - containerPort: 9090
+                      name: metrics
                   env:
+                    - name: METRICS_DOMAIN
+                      value: knative.dev/serving
```

Comment:
do we need something like this as well for the `knative-operator`?

Reply:
Ah, I think it's here: https://github.com/openshift-knative/serverless-operator/blob/a3b7c60f8542f8c232093d4abf0a2d7b81cd087b/olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml#L380-L386